sympy/core/symbol.py

```diff
@@ -205,21 +205,56 @@ def numbered_string_incr(s, start=0):
 
 class Symbol(AtomicExpr, Boolean):
     """
+    Symbol class is used to create symbolic variables.
+
+    Explanation
+    ===========
+
+    Symbolic variables are placeholders for mathematical symbols that can represent numbers, constants, or any other mathematical entities. A symbolic variable, created using the 'Symbol' class, can be used in mathematical expressions and perform symbolic computations.
```

Comment:
Unfortunately, I think that we need better prompt to explain symbols, 
because the definition just seems like cyclic 

> **Symbolic** variables are placeholders for mathematical **symbols**

Reply:
Ok👍